src/doc/src/reference/resolver.md

```diff
@@ -201,6 +203,87 @@ ecosystem if you publish a SemVer-incompatible version of a popular library.
 [semver trick]: https://github.com/dtolnay/semver-trick
 [`downcast_ref`]: ../../std/any/trait.Any.html#method.downcast_ref
 
+### Rust version
+
+To support developing software with a minimum supported [Rust version],
+the resolver can take into account a dependency version's compatibility with your Rust version.
+This is controlled by the config field [`incompatible-rust-version`].
+
+With the `fallback` setting, the resolver will prefer packages with a Rust version that is
+equal to or greater than your own Rust version.
+For example, you are using Rust 1.85 to develop the following package:
+```toml
+[package]
+name = "my-cli"
+rust-version = "1.62"
+
+[dependencies]
+clap = "4.0"  # resolves to 4.0.32
+```
+The resolver would pick 4.0.32 as its Rust version is 1.60.0.
+- 4.0.0 also has a Rust version of 1.60.0 but that is a [lower version number](#version-numbers)
+- 4.5.20 is a much [higher version](#version-numbers) and it has a Rust version of 1.74.0 which is compatible with your 1.85 toolchain but it is incompatible with `my-cli`'s Rust version of 1.62
```

Comment:
This feels more cohesive to me.

```suggestion
The resolver would pick version 4.0.32 for dependency `clap` because
- 4.0.32 has a Rust version of 1.60.0, which is compatible with `my-cli`'s Rust version of 1.62.
- 4.0.0 also has a Rust version of 1.60.0 but that is a [lower version number](#version-numbers).
- Despite 4.1.0 and later versions all have a much [higher version](#version-numbers), their Rust versions are incompatible with `my-cli`'s Rust vesion. For example, 4.5.20 has a Rust version of 1.74.0 which is compatible with your 1.85 toolchain, but not with `my-cli`'s Rust version of 1.62.
```

Reply:
Note that that is mixing sentence structures.  The header was changed to end with "because" and "because despite" doesn't make sense to say.  I'm working on my own teak.

Reply:
Thanks! Looks nicer :)